app/assets/javascripts/discourse/lib/to-markdown.js.es6

```diff
@@ -126,6 +126,53 @@ export class Tag {
     };
   }
 
+  static aside() {
+    return class extends Tag {
+      constructor() {
+        super("aside");
+        this.gap = "\n\n";
+      }
+
+      decorate(text) {
+        const parent = this.element.parent;
+
+        if (this.name === "p" && parent && parent.name === "li") {
+          // fix for google docs
```

Comment:
What's the issue with Google Docs?

Reply:
I do not know. I needed to extend the class that is returned by `Tag.block('aside')` and did not realize that I could call that method with `extend`, so I just copied that class and modified it.